security/cert_manager_operator/cert-manager-operator-issuer-acme.adoc

```diff
@@ -6,7 +6,24 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-The {cert-manager-operator} supports using ACME CA servers, such as Let's Encrypt, to issue certificates.
+The {cert-manager-operator} supports using Automated Certificate Management Environment (ACME) CA servers, such as Let's Encrypt, to issue certificates. Explicit credentials are configured by specifying the secret details in the `Issuer` API object. Ambient credentials are extracted from the environment, metadata services, or local files which are not explicitly configured in the `Issuer` API object.
+
+[NOTE]
+====
+To configure an ACME issuer to solve DNS-01 challenges by using explicit and ambient credentials, you can use the `ClusterIssuer` object instead of the `Issuer` object.
```

Comment:
> you can use the `ClusterIssuer` object instead of the `Issuer` object

This "instead of" seems confusing for me. For explicit, users can use Issuer. For ambient, users can't use Issuer by default but still can use it if cert-manager is configured with --issuer-ambient-credentials.

Reply:
Addressed. Added two separate notes for clarity

Reply:
https://github.com/openshift/openshift-docs/pull/59702/files#diff-1fa7438f3b35bcfae2a4c11d374f7c12ecdf578134a7cf3689d2d5ea6c4f7726R11-R31